network.json

```diff
@@ -46,6 +46,16 @@
 					"format": "^((25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])(\\\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])){3})?$",
 					"title": "Gateway",
 					"showSummary": true
+				},
+				"ipv6Subnet": {
+					"type": "string",
```

Comment:
Thanks for the changes @gkvijay. 
Can you please add a format field for ipv6 subnet and gateway fields?
schema compiler will generate validators based on format specification..